From 89cc49a60c101a40df8d8b23a45ed68b23dbcf4b Mon Sep 17 00:00:00 2001
From: Uldiniad <olivercscott@gmail.com>
Date: Wed, 4 Jul 2018 20:44:27 -0400
Subject: [PATCH 09/19] Settings: Remove battery percentage switch

The setting is available in Status bar settings.
Remove duplicate to improve UX.

Change-Id: If4cfdb8289d3712b111f3d65b70f3d790fb83923
---
 .../settings/display/BatteryPercentagePreferenceController.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/com/android/settings/display/BatteryPercentagePreferenceController.java b/src/com/android/settings/display/BatteryPercentagePreferenceController.java
index 449315873c..0a4892693f 100644
--- a/src/com/android/settings/display/BatteryPercentagePreferenceController.java
+++ b/src/com/android/settings/display/BatteryPercentagePreferenceController.java
@@ -40,7 +40,7 @@ public class BatteryPercentagePreferenceController extends AbstractPreferenceCon
 
     @Override
     public boolean isAvailable() {
-        return true;
+        return false;
     }
 
     @Override
-- 
2.17.1

